src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+function helper(obj) {
```

Comment:
The function should be called according it's functionality.
There can be lots of helper functions that just help to simplify the code.